browser/components/MarkdownPreview.js

```diff
@@ -255,6 +261,10 @@ export default class MarkdownPreview extends React.Component {
       el.addEventListener('click', this.checkboxClickHandler)
     })
 
+    _.forEach(this.refs.root.contentWindow.document.querySelectorAll('a'), (el) => {
+      el.addEventListener('click', this.linkClickHandler)
```

Comment:
You can write this line at [this line](https://github.com/BoostIO/Boostnote/pull/336/files#diff-a54dec67725b3e604a34d39a7fcb72f6R257).